frontend/prettier.config.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import("prettier").Config} */
```

Comment:
Is there a reason you want to move this to a separate file?

Reply:
https://github.com/prettier/prettier/issues/15085
When i added the plugin in the `package.json` file, prettier wasn't able to find the package. I had to pass in the path to the plugin. So i moved to a separate file.

Reply:
I changed the path to a relative one, and it seems to work.